lambda/MAS.Tests/IntergrationTests/ContentControllerTests.cs

```diff
@@ -1,33 +1,43 @@
 using System.Threading.Tasks;
 using Xunit;
-using Newtonsoft.Json;
 using MAS.Tests.Infrastructure;
-using System.Net.Http;
-using System.Text;
-using System.Net;
 using Shouldly;
+using Amazon.S3;
+using Amazon;
 using MAS.Configuration;
+using System.IO;
 
 namespace MAS.Tests.IntergrationTests
 {
     public class ContentControllerTests : TestBase
     {
-        public ContentControllerTests()
-        {
-            AppSettings.AWSConfig = TestAppSettings.GetAWSConfig();
-        }
-
         [Fact]
-        public async Task Put()
+        public async Task PutCMSItemIntoS3()
```

Comment:
Which controller is this testing? On the face of it, it seems it is testing third part code.

Is it testing: /api/content

Should the test make it more clear what it's actually testing?

Reply:
Renamed test and namespace